inc/targetbase.class.php

```diff
@@ -1179,7 +1267,7 @@ public function prepareInputForUpdate($input) {
          if (!$target->isNewItem()) {
             $target->update([
                'id' => $target->getID(),
-               'name' => $input['name'],
+               'name' => $DB->escape($input['name']),
```

Comment:
Are you sure that $input is not already escaped by GLPI ? What happens if you have a backslash in $input['name'] ?